examples/EU DST example

```diff
@@ -0,0 +1,59 @@
+/* EU DST example */
+/* version 3: dd 14-10-2024 */
```

Comment:
This date format is ambiguous. Prefer the ISO date format: 2024-10-14. Or, better yet, remove the line: the history of this file is in the git repository anyways.